site.yml

```diff
@@ -14,6 +14,18 @@
   - { role: base }
   - { role: serf }
   - { role: dev }
+  - { role: vagrant }
+
+- hosts: build
```

Comment:
I addressed it by using a different tag `prebake-for-test` for vagrant, vbox and packer tasks that way we can isolate build images (contiv/build) from lab hosts (contiv/lab) by using these tags differently.